debian_script.bsh

```diff
@@ -1,27 +1,49 @@
 #!/usr/bin/env bash
 
-set -eu
+set -e
+
+ARCH=amd64
+case "$(uname -m)" in
+  arm64|aarch64)
+    ARCH=arm64;;
+esac
 
 REPO_DIR=${REPO_DIR:-/repo}
 GIT_LFS_BUILD_DIR=${GIT_LFS_BUILD_DIR:-/tmp/docker_run/src/git-lfs}
 SRC_DIR=${SRC_DIR:-/src}
 REPO_CODENAME=${REPO_CODENAME:-$(source /etc/os-release; echo $VERSION | sed -r 's|.*\((.*)\)|\1|')}
 
+process_architecture () {
+  local arch="$1"
+  local arch_arg="$2"
+
+  cp -r -T "${SRC_DIR}" "${GIT_LFS_BUILD_DIR}/$arch"
+  cd "${GIT_LFS_BUILD_DIR}/$arch"
+  git clean -xdf .
+  git checkout-index --force --all
+  dpkg-buildpackage -d -us -uc -b $arch_arg
```

Comment:
Could this just use `-a"$arch"` for all builds, and then there'd be no need for a separate optional argument (and `set -u` could stick around)?  In other words, if specifying `-aamd64` or `-aarm64` would work, maybe it's easiest to just do that, and then there's no need to pass a somewhat magic `-a$arch` argument to this function.

If there still needs to be a second argument, though, I might suggest having it at least not include the `-a` part, and have this function check if the second argument is blank or unset, and if not, construct the `-a"$2"` part itself, so the caller doesn't need to know about the `-a` part.  It's a minor thing, really, though.

Reply:
Sure, we can do that.